continuous-integration/linux/install-sdist.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+SCRIPT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+ROOT_DIR=$(realpath "$SCRIPT_DIR/../..")
+
+# Install source distribution
+pip install --upgrade pip
```

Comment:
Missing: `|| exit $?`

Reply:
Could you also run `python(3?) -m pip install --upgrade pip`?

Reply:
Yeah that was missing, thanks.

Regarding pip: I followed the convention from build.sh, where we use straight forward pip commands. These agents only have one pip installed. Do you have a good argument for complicating the command?

Reply:
This [link](https://snarky.ca/why-you-should-use-python-m-pip/) explains it nicely

Reply:
Alright I'll change it across the board then.